lib/experimental/Widgets/ChartWidgetEmptyState/index.tsx

```diff
@@ -0,0 +1,90 @@
+import { Button, ButtonProps } from "@/components/Actions/Button"
+import { IconType } from "@/components/Utilities/Icon"
+import { cn } from "@/lib/utils"
+import { Card, CardContent, CardHeader, CardTitle } from "@/ui/card"
+import { forwardRef } from "react"
+import EmptyBarChart from "./Backgrounds/EmptyBarChart"
+import EmptyLineChart from "./Backgrounds/EmptyLineChart"
+
+type Type = "bar-chart" | "line-chart"
+
+export interface Props {
+  title: string
+  content: string
+  buttonLabel?: string
+  buttonIcon?: IconType
+  buttonAction?: () => void
+  type: Type
+}
+
+const ROOT_CLASSNAMES: Record<Type, string> = {
+  "line-chart": "border-f1-border-accent-alpha20",
+  "bar-chart": "border-f1-border-promoteAlpha30",
+}
+
+const CONTENT_CLASSNAMES: Record<Type, string> = {
+  "line-chart": "min-h-48",
+  "bar-chart": "min-h-64",
+}
+
+const BG_CLASSNAMES: Record<Type, string> = {
+  "line-chart": "from-f1-background-accent-alpha5",
+  "bar-chart": "from-f1-background-promote-alpha5",
+}
+
+const BUTTON_VARIANT: Record<Type, ButtonProps["variant"]> = {
+  "line-chart": "default",
+  "bar-chart": "promote",
+}
+
+export const ChartWidgetEmptyState = forwardRef<HTMLDivElement, Props>(
+  function WidgetEmptyState(
+    { title, content, buttonLabel, buttonIcon, buttonAction, type },
+    ref
+  ) {
+    const rootClassName = ROOT_CLASSNAMES[type]
+    const contentClassName = CONTENT_CLASSNAMES[type]
+    const bgClassName = BG_CLASSNAMES[type]
+    const buttonVariant = BUTTON_VARIANT[type]
+
+    return (
+      <Card
+        className={cn(
+          "relative flex gap-4 overflow-hidden border-dashed",
+          rootClassName
+        )}
+        ref={ref}
+      >
+        <CardHeader className="-mr-1 -mt-1">
```

Comment:
is it necessary? Maybe we should reconsider the card style?

Reply:
Yep, some adjustment is needed, according to figma the card at the top and on the right has less spacing